src/hbscorez/settings.py

```diff
@@ -129,6 +129,14 @@
             'class': 'logging.FileHandler',
             'filename': ROOT_DIR / 'hbscorez.log',
             'formatter': 'verbose',
+            'encoding': 'UTF-8'
+        },
+        'file': {
+            'level': 'INFO',
+            'class': 'logging.FileHandler',
+            'filename': ROOT_DIR / 'hbscorez_INFO.log',
+            'formatter': 'verbose',
+            'encoding': 'UTF-8'
```

Comment:
I'll have to check if the production settings extends or overwrite the logging configuration.
Also I'm not sure wether this should really be the default config, or if it's just personal preference.

Reply:
The idea was to split the logging into different files, which retrospecively spoken, does not improve anything. However the encoding could be added due to some german "Umlaute" appearing in the association, district and / or team names. New PR is in the pipeline :)